pkgs/development/ocaml-modules/parmap/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildDunePackage, fetchFromGitHub }:
+
+buildDunePackage rec {
+  pname = "parmap";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "rdicosmo";
+    repo   = pname;
+    rev    = version;
+    sha256 = "0xpy2012i8ckpii4cqxwcr5v7d4719md7y6rfdqyv2xy8nq0qhrf";
+  };
```

Comment:
Please use the release archive from <https://github.com/rdicosmo/parmap/releases/download/1.1/parmap-1.1.tbz>. It has correct version information.

Reply:
OK, done.